### DISCUSSION TITLE
Parameters on schedule workflow

### DISCUSSION THREAD

**Discussion by andresmafra** [community/community#21424] [Other Feature Feedback, Questions, & Ideas]
<p>Is it possible to include parameters in a schedule GHA call? Example:</p>
<pre><code class="lang-auto">on:
  schedule:
    - cron: '0 6 * * *'
       var:
       default: 'MY_VAR'
    - cron: '0 9 * * *'
       var:
       default: 'MY_OTHER_VAR'
</code></pre>

**Comment by ossanna16** [comment, marked as answer]
<p>Hi thank you for your post! This is not a GitHub related post so you may find more help in more general forums like Stackoverflow.</p>

**Reply by dhimmel** [reply]
> This is not a GitHub related post

Hmm, pretty sure the question is about GitHub actions and the `schedule` trigger.

@andresmafra noting that I suggested a related enhancement at https://github.com/community/community/discussions/25662#discussioncomment-3248650 to allow naming each cron schedule condition. The discussion there might help you with a solution here by accessing the `github.event.schedule` context.

**Reply by andresmafra** [reply]
Cool @dhimmel, I think it helps a lot.
